boards/arm/nucleo_h743zi/nucleo_h743zi.dts

```diff
@@ -112,7 +113,52 @@
 		     &eth_txd1_pb13>;
 };
 
+
 &spi1 {
 	status = "okay";
 	pinctrl-0 = <&spi1_sck_pa5 &spi1_miso_pa6 &spi1_mosi_pb5>;
 };
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * https://docs.zephyrproject.org/latest/reference/devicetree/index.html#fixed-flash-partitions
```

Comment:
This link is obsolete (I should add again, as this is at least the 2nd time the link used for this doc changes).
Let's remove the whole comment as this is now a common feature and there is no specific reason to comment it more than other dts pieces.
We might need to clean this up from other boards as well.